src/Reader/StandaloneExtensionManager.php

```diff
@@ -49,4 +49,30 @@ public function get($extension)
         $class = $this->extensions[$extension];
         return new $class();
     }
+
+    /**
+     * Add an extension.
+     *
+     * @param string $name
+     * @param string $class
+     */
+    public function add($name, $class)
+    {
+        $this->extensions[$name] = $class;
+    }
+
+    /**
+     * Remove an extension.
+     *
+     * @param string $name
+     * @return bool
```

Comment:
It does not need to report the result. It is idempotent operation. void would be fine